### DISCUSSION TITLE
Skipping duplicate collection

### DISCUSSION THREAD

**Discussion by Sir-Gavin** [meisnate12/Plex-Meta-Manager-Configs#53] [General]
after some errors and clearing the plex tags 
im tring to run the script againg but for the movies im getting 
Config Warning: Skipping duplicate collection: EXTRA 
how can i rerun complete scans again
i clearded all the logs and deleted the cache file from the appdate folder  that didnt work

**Comment by meisnate12** [comment]
you have two collections with the same name

**Comment by Sir-Gavin** [comment]
ok sorry to sound like a idiot you helped me clear all the tags 
what do i do the clear all the collections and sharing labels
 so i can start over fresh 
thanks
